.github/workflows/nodejs.yml

```diff
@@ -5,7 +5,7 @@ on: [push]
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
This is because of a Chromium bug, see https://github.com/puppeteer/puppeteer/issues/12818

Reply:
I'll subscribe to that bug. Ideally we can re-instate `ubuntu-latest` at some point.

Reply:
Actually, that bug is closed. I'll go hunt for a bug about the override for `runs-on`.

Reply:
I think that was closed because its really a Chromium bug which is here: https://issues.chromium.org/issues/373753919